File: android/entities/src/main/java/com/tiagohs/entities/contents/ContentVideo.kt
package com.tiagohs.entities.contents

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class ContentVideo(

    @SerializedName("video_id")
    val videoId: String,

    @SerializedName("height")
    val height: Int? = null,

    @SerializedName("information")
    val information: ContentInformation
): Content(), Serializable

File: ios/HistoryCinema/HistoryCinema/Entities/LocalFiles/Contents/ContentVideo.swift
//
//  ContentVideo.swift
//  HistoryCinema
//
//  Created by Tiago Henrique da Silva on 21/02/21.
//

import Foundation
import ObjectMapper

class ContentVideo: Content {
    var videoId: String!
    var height: Int?
    var information: ContentInformation!
    
    required init?(map: Map) {
        super.init(map: map)
    }

    override func mapping(map: Map) {
        super.mapping(map: map)
        
        videoId         <- map["video_id"]
        height         <- map["height"]
        information         <- map["information"]
    }
}
